crypto/include/internal/modes_int.h

```diff
@@ -1,34 +1,219 @@
 /*
- * Copyright 2019 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2010-2019 The OpenSSL Project Authors. All Rights Reserved.
  *
  * Licensed under the Apache License 2.0 (the "License").  You may not use
  * this file except in compliance with the License.  You can obtain a copy
  * in the file LICENSE in the source distribution or at
  * https://www.openssl.org/source/license.html
  */
 
+/* TODO(3.0) Move this header into provider when dependencies are removed */
```

Comment:
It really needs to be refactored, and that should be mentioned too.  I *insist* that there are parts of this that really should be private to the modes sub-system.

Reply:
Ignore that...

I dug into the modes stuff and realised that while it would be *nice* if the `modes_lcl.h` stuff was `crypto/modes` private, it turns out not to be.  So errr never mind and sorry for all the noise.